packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -245,12 +246,12 @@ const helpers = {
   },
 
   negotiateOptionsForInvalidRequest(httpResponses: IHttpOperationResponse[]): IHttpNegotiationResult {
-    // currently only try to find a 400 response, but we may want to support other cases in the future
-    const code = '400';
-    const response = findResponseByStatusCode(httpResponses, code);
-    // TODO: what if no 400 response is defined?
+    const response =
+      findResponseByStatusCode(httpResponses, '422') ||
+      findResponseByStatusCode(httpResponses, '400') ||
+      createResponseFromDefault(httpResponses, '422');
```

Comment:
@philsturgeon Here are the relevant bits. Invalid request:

Find 422 -> Find 400 -> Find a default -> Generate a 422 automagically.